docs/user-guide/index.md

```diff
@@ -8,7 +8,7 @@ The Kubernetes **Guides** can help you work with various aspects of the Kubernet
 
 * The Kubernetes [User Guide](#user-guide-internal) can help you run programs and services on an existing Kubernetes cluster.
 * The [Cluster Admin Guide](/docs/admin/) can help you set up and administrate your own Kubernetes cluster.
-* The [Developer Guide](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel) can help you either write code to directly access the Kubernetes API, or to contribute directly to the Kubernetes project.
+* The [Developer Guide] can help you either write code to directly access the Kubernetes API, or to contribute directly to the Kubernetes project.
```

Comment:
Is there a particular reason why you are using the reference syntax for the link instead of just having the URL next to the link? If not, I would suggest keeping it next to the link definition for the sake of readability for future reviewers, i.e.:

The \[Developer Guide](https://github.com/kubernetes/community/blob/master/contributors/devel/README.md) can help you either write code to directly access the Kubernetes API, or to contribute directly to the Kubernetes project.